scripts/tailwind/input.css

```
@@ -1,3 +1,31 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* Toast slide animations. ui/js/toast.js assigns the enter one inline and
+   declares the exit one in data-animation-out, which <inline-alert> plays
+   on removal. */
+@keyframes slideInRight {
```

Stylelint (17.14.0)

[error] Expected keyframe name "slideInRight" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)